src/URLValidator.js

```diff
@@ -0,0 +1,31 @@
+const {
+  noTextError,
+  noToCaseError,
+  invalidToCaseError,
+} = require('./constants');
+
+const toCases = [
+  'SNAKE',
+  'KEBAB',
+  'CAMEL',
+  'PASCAL',
+  'UPPER',
+];
+
+const getURLErrors = (textToConvert, toCase) => {
+  const errors = [];
+
+  if (!textToConvert) {
+    errors.push(noTextError);
+  }
+
+  if (!toCase) {
+    errors.push(noToCaseError);
+  } else if (!toCases.includes(toCase)) {
+    errors.push(invalidToCaseError);
+  }
+
+  return errors;
```

Comment:
better to write separate if blocks
```suggestion
  if (!toCase) {
    errors.push(noToCaseError);
  }
  
  if (!toCases.includes(toCase) && toCase) {
    errors.push(invalidToCaseError);
  }

  return errors;
```